UITestHelpers/XCUIElement+Extensions.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+
+extension XCUIElement {
+
+    public func paste(text: String) -> Void {
```

Comment:
Had to change `private func...` to `public func...` to make is usable outside the extension

Reply:
Thank you! While the concept of access control looks very straightforward on the surface, for me it gets complicated when getting deeper into details. Mostly, I feel, because of the lack of practice with access control. I keep returning to this page from time to time, and try to understand it more than at the previous time.

Reply:
>  I keep returning to this page from time to time, and try to understand it more than at the previous time.

That's it with most things really. It takes time and practice.